packages/devtools/pubspec.yaml

```diff
@@ -18,8 +18,12 @@ environment:
   sdk: '>=2.3.0 <3.0.0'
 
 dependencies:
-  devtools_server: ^0.1.13
+#  devtools_server: ^0.1.14
+  devtools_server:
```

Comment:
use the dependency overrides section instead as a general practice. that avoids cases in the future where package resolution gets confused by local dependencies.